src/background/background.js

```
@@ -51,33 +52,37 @@ chrome.commands.onCommand.addListener((command) => {
 // Listen for tab updates to switch popups
 chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   if (changeInfo.url) {
-    if (tab.url.includes('bigmachines.com/admin/ui/branding/edit_header_footer.jsp')) {
+    if (tab && tab.url && tab.url.includes('bigmachines.com/admin/ui/branding/edit_header_footer.jsp')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)